poms/bom-test/pom.xml

```diff
@@ -178,6 +178,17 @@
                 <artifactId>camel-quarkus-integration-tests-support-custom-type-converter</artifactId>
                 <version>${camel-quarkus.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.quarkus</groupId>
+                <artifactId>camel-quarkus-integration-tests-support-splunk</artifactId>
+                <version>${camel-quarkus.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.camel.quarkus</groupId>
+                <artifactId>camel-quarkus-integration-tests-support-splunk</artifactId>
+                <version>${camel-quarkus.version}</version>
+                <type>test-jar</type>
```

Comment:
It's unfortunate that just because of the constants, the dependency has to be added for the second time with the test scope. I think that overall this it is a nicer solution then having redefine the constants in the runtime for the second time.